pulp_container/app/models.py

```diff
@@ -512,6 +517,35 @@ class Meta:
             ),
         ]
 
+    def model_to_dict(self):
```

Comment:
Did not `from django.forms.models import model_to_dict` suit our needs?

Reply:
I think it does, but when I tried to use it I had 3 concerns:
* the "old version" was returning a field called `pulp_domain` whereas with django's model_to_dict it returns `pulp_domain_id`, so we would need to somehow handle this corner case
* with django's model_to_dict we would still need _get_pull_through_remote_data (not the method itself, but the `IGNORED_PULL_THROUGH_REMOTE_ATTRIBUTES` that is used by it)
* had to remove the url from `IGNORED_PULL_THROUGH_REMOTE_ATTRIBUTES` (it was causing an exception without it)

Here is what I tried:
```
    remote_data = model_to_dict(pull_through_remote, exclude=IGNORED_PULL_THROUGH_REMOTE_ATTRIBUTES)
    remote_data['pulp_domain_id'] = remote_data['pulp_domain']
    remote_data.pop('pulp_domain', None)
```

Do you think we should use it instead?

Reply:
Did the execution fail when you passed "pulp_domain_id" to "Remote.create"? Either way, we can try doing what you have suggested:
```
IGNORED_PULL_THROUGH_REMOTE_ATTRIBUTES = ("includes", "excludes", "pulp_domain")
remote_data = model_to_dict(pull_through_remote, exclude=IGNORED_PULL_THROUGH_REMOTE_ATTRIBUTES)
```